lib/index.js

```diff
@@ -1,120 +1,133 @@
-'use strict';
```

Comment:
I'm pretty sure you don't have to wrap it in a closure and that browserify will do it for us